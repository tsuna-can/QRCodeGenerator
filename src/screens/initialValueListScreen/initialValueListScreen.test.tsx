import React from 'react';
import renderer from 'react-test-renderer';
import InitialValueListScreen from './initialValueListScreen';

describe('HomeScreen', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<InitialValueListScreen navigation={undefined} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
