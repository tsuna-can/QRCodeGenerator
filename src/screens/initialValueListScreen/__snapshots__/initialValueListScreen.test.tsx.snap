// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListScreen renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "flex": 1,
      "margin": 10,
    }
  }
>
  <RCTScrollView>
    <View />
  </RCTScrollView>
</RCTSafeAreaView>
`;
